### PR TITLE
test(node/path): Enable test-path-exists.js

### PR DESCRIPTION
**Pull request by wafuwafu13**

### CODE REVIEW

**Comment by wafuwafu13** [comment]
https://github.com/denoland/deno_std/runs/5180211597?check_suite_focus=true
failures native modules are extensible

https://github.com/denoland/deno_std/runs/5244481502?check_suite_focus=true
same problem as https://github.com/denoland/deno_std/pull/1882#discussion_r800139492

File: node/module_all.ts [hunk 1]
```diff
@@ -116,6 +116,8 @@ export default {
   "internal/util": internalUtil,
   net,
   os,
+  "path/posix": path,
```
Comment (kt3k) [inline node/module_all.ts:119]: I think we need to pass something like the below here:

```ts
import pathPosix from "./path/posix.ts";
```
And that means we need to create default export of `path/posix.ts`

```ts
export default {
  sep,
  delimiter,
  ...
}
```
We probably also need to update `path/mod.ts` like the below

```diff
-import * as _posix from "./posix.ts";
+import _posix from "./posix.ts";
```

**Review by kt3k — COMMENTED** [review]
Sorry for missing this PR. Left a comment

**Review by kt3k — APPROVED** [review]
@wafuwafu13 LGTM! Nice work!

**Pull request merged**